### ISSUE TITLE
Deamon creation

### ISSUE THREAD

**Issue by show0k** [poppy-project/poppy-daemon#1]
Wouldn't be a better idea to use a dedicated library to create the daemon ?
[Daemonocle](https://github.com/jnrbsn/daemonocle) seems used (I found it on a reddit thread on using Flask as a desktop GUI). Furthermore, it implements some high level [good features](https://github.com/jnrbsn/daemonocle#the-problem).

**Comment by pierre-rouanet** [comment]
Yes I agree. Unfortunately the few libraries I tried where not working great... Let me know if this one is doing the job!
